www/src/components/pagination/index.js

```diff
@@ -8,7 +8,7 @@ import { options, rhythm } from "../../utils/typography"
 
 class Pagination extends React.Component {
   changePage = e => {
-    navigate(`/blog/${e.target.value}`)
+    navigate(`/blog/page/${e.target.value}`)
```

Comment:
I'm not sure - this might not work for first page as well? (special case of it being just `/blog` with page number)?

Reply:
Since the value for the first page is an empty string (which is falsy)
```diff
- navigate(`/blog/${e.target.value}`)
+ navigate(e.target.value ? `/blog/${e.target.value}` : `/blog`)
```